src/main/java/de/bonndan/nivio/input/external/springboot/CompositeHealthDeserializer.java

```diff
@@ -0,0 +1,27 @@
+package de.bonndan.nivio.input.external.springboot;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.node.TextNode;
+import org.springframework.boot.jackson.JsonComponent;
+
+import java.io.IOException;
+import java.util.Map;
+
+@JsonComponent
+public class CompositeHealthDeserializer extends JsonDeserializer<JsonCompositeHealth> {
+
+    @Override
+    public JsonCompositeHealth deserialize(JsonParser jsonParser, DeserializationContext deserializationContext) throws IOException {
```

Comment:
There is no test for this, is there? Also I didn't find one for the `JsonCompositeHealth`. Why not addind directly a test for this new functionality? The test should be rather simple now.

I started adding jacoco with https://github.com/dedica-team/nivio/pull/329 and from what I saw, we can really improve on test code coverage!

Reply:
It's tested indirectly in `SpringBootHealthHandlerTest`. Although you're right, I am not very motivated to invest my time in an JsonParser/TreeNode/bla etc. deep dive.